cmd/suggestion/random/v1alpha1/Dockerfile

```diff
@@ -2,7 +2,12 @@ FROM golang:alpine AS build-env
 # The GOPATH in the image is /go.
 ADD . /go/src/github.com/kubeflow/katib
 WORKDIR /go/src/github.com/kubeflow/katib/cmd/suggestion/random
-RUN go build -o random ./v1alpha1
+RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+        apk --update add gcc musl-dev  && \
```

Comment:
@dreamryx good job.
do you miss katib/cmd/suggestion for v1alpha2?

Reply:
OK. I will re-commit docker file for v2.